CHANGELOG.md

```diff
@@ -2,6 +2,124 @@
 
 ## 2015-04-20, Version 1.8.1, @chrisdickinson
 
+### Breaking changes
+
+Full details at https://github.com/iojs/io.js/wiki/Breaking-Changes#200-from-1x
+
+* V8 upgrade to 4.2, minor changes to C++ API
+* `os.tmpdir()` is now cross-platform consistent and will no longer returns a path with a trailling slash on any platform
+* While not a *breaking change* the 'smalloc' module has been deprecated in anticipation of it becoming unsupportable with a future upgrade to V8 4.4. 
```

Comment:
@silverwind, @bnoordhuis: happy with this mention of smalloc in "Breaking changes"?